ci/mac_ci_steps.sh

```diff
@@ -4,7 +4,8 @@ set -e
 
 . "$(dirname "$0")"/setup_gcs_cache.sh
 
-BAZEL_BUILD_OPTIONS="--curses=no --show_task_finish --verbose_failures ${BAZEL_BUILD_EXTRA_OPTIONS}"
+BAZEL_BUILD_OPTIONS="--curses=no --show_task_finish --verbose_failures ${BAZEL_BUILD_EXTRA_OPTIONS} \
+  --action_env=PATH=/usr/bin:/usr/local/bin:/bin:/usr/sbin"
```

Comment:
What's the reasoning for each of those directories? Why do we need `/usr/sbin`? Don't we need `/bin` as well?

Also, at the very least it's missing `/opt/local/bin` for MacPorts (not everybody is using Homebrew).

Reply:
`/bin` is in there. 

As for `/usr/sbin`: I had to add that locally because `sysctl` was being called, unfortunately I don't recall what was actually calling that. Haven't verified that this is required in CI.

Are people actually using this file for building locally? I can add it `/opt/local/bin`, but I thought this was meant for running in CI which just uses brew?

Reply:
For context this is breaking the master CI build (and branch builds), updated the description to reflect that.

Reply:
Ops, `/bin` is indeed there, I totally missed it. Thanks. Could you sort them for readability, by any chance? i.e. `PATH=/usr/local/bin:/opt/local/bin:/usr/bin:/bin`.

`/usr/sbin/sysctl` is called on macOS in `ci/build_container/build_and_install_deps.sh`, but perhaps it makes more sense to use the full path there instead of adding `/usr/sbin/` to `$PATH`? I'm running local build to verify that this works.

wrt `/opt/local/bin`, I honestly don't know if people use stuff in `ci/` for local builds (I don't), but since we're fixing it for Homebrew, and the fix for MacPorts is basically free, I don't see a reason not to do it, really.